src/client/components/dashboard/CreateNewModal.tsx

```diff
@@ -128,7 +128,7 @@ export const CreateNewModal: FC<CreateNewModalProps> = ({ onClose }) => {
     createChecker.mutate({
       ...base,
       ...data,
-      isActive: false,
+      isActive: true,
```

Comment:
This should be enforced in the backend publish endpoint, instead of relying on the client passing `isActive` to true. In addition, the expected behaviour is that if the checker is already published and set to inactive, republishing it should not make it active immediately.